### DISCUSSION TITLE
Moosedocs gallery and cards broken

### DISCUSSION THREAD

**Discussion by hugary1995** [idaholab/moose#16275] [Q&A Tools] [answered]
I updated my moose and conda environment just now. The doc website for my app looks different from before.
It looks like this before:
![before](https://user-images.githubusercontent.com/31555204/99892187-4ad60080-2c40-11eb-8d74-d3c6896b3681.png)

However, after the update, it looks like some padding are added before the gif in the card:
![after](https://user-images.githubusercontent.com/31555204/99892185-43165c00-2c40-11eb-9ee9-db7fd41ea747.png)

By the way, I think there should be a Q&A Moosedocs category.

**Comment by aeslaughter** [comment]
What App?


FYI, the "Q&A Tools" is for this sort of thing.

**Reply by hugary1995** [reply]
Here is the corresponding markdown file:
https://github.com/hugary1995/raccoon/blob/master/doc/content/index.md

**Comment by aeslaughter** [comment, marked as answer]
This should fix it: #16281
